.circleci/config.yml

```diff
@@ -115,17 +73,15 @@ workflows:
   # Workflow which runs on each push
   build_test_deploy_on_push:
     jobs:
-      - build_and_test_python27
       - build_and_test_python36
       - deploy:
           requires:
-            - build_and_test_python27
+            - build_and_test_python36
```

Comment:
Might be a good idea to include a few comment lines about specifics of py2 vs py3 support for this pack.

Reply:
I added a note to the README.